### PR TITLE
Add Flutter Appwrite Auth System and Flutter Appwrite Todo

### PR DESCRIPTION
**Pull request by devkishor8007**
[Add Flutter Appwrite Auth System  and Flutter Appwrite Todo](https://github.com/devkishor8007/awesome-appwrite/commit/1a247b6251c7a28a54cea030361e0e7a6fc65571)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,6 +164,7 @@ Submit your projects and win some Awesome Appwrite Swag 🤩
 
 * **Godot + Appwrite** - A demo app to showcase Godot's integration with Appwrite. [Source Code](https://github.com/lohanidamodar/godot-appwrite)
 
+
```
Comment (christyjacob4) [inline README.md:167]: You can remove this line

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Please address the comments

File: README.md (hunk 1 above)
Comment (devkishor8007) [reply README.md:167]: [Please Do check the latest once](https://github.com/appwrite/awesome-appwrite/pull/79/commits/1a247b6251c7a28a54cea030361e0e7a6fc65571)

I added two PR

- Flutter Appwrite Auth System
- Flutter Appwrite Todo

Comment (christyjacob4) [reply README.md:167]: Nope. This extra line is still present

Comment (devkishor8007) [reply README.md:167]: In167 Line - There is no + sign 
Please Look this screenshot, 

image

I have checked the GitHub ReadMe.md and also my VSC clone file, but I have not got any +.

What should I do?

Comment (devkishor8007) [reply README.md]: [Now I remove the 167 line](https://github.com/devkishor8007/awesome-appwrite/commit/78336bc14c3a1820ef9185b3450be393b8b5e292)

Make me correct If I am wrong?
Thank you.

**Review by christyjacob4 — APPROVED** [review]

**Review by dj3364 — APPROVED** [review]

**Pull request merged**